### ISSUE TITLE
Temporal Consistency

### ISSUE THREAD

**Issue by JonasFrey96** [leggedrobotics/wild_visual_navigation#198]
I now checked the temporal consistency and found a pytorch_geometric bug. 
When loading to different graphs by the data loader the collate_fn does not collate them correctly but uses the ptr available from the first graph. This led to a mismatch within the correspondence matching. 
I simplified the data loaders made them faster and removed the default parameters. 
Additionally, the traversability loss should be now simpler. 

Important:
1. The online mode without the correspondences should be still working but needs to be verified.

Verify Ablations
1. all ablations training scripts
2. all ablations visu scripts
3. train
4. node

Ablate the temporal consistency constraint loss
1. Hopefully the results look now better
